### ISSUE TITLE
LOSS FUNCTION TO TRAIN

### ISSUE THREAD

**Issue by cvarun94** [ai-forever/Real-ESRGAN#7]
Hi !
Thanks alot for this repo.. I wish to train realSRGAN on custom data using Pytorch Lightning. And for that I require the LOSS function and optimizer details. Would request you to provide that.

Thank You
Varun

**Comment by kelisiya** [comment]
+1 , and i want to know the dataset domain

**Comment by anaguilarar** [comment]
Hi, is it possible to access any documentation related to the training process?

**Comment by phineas-pta** [comment]
https://github.com/xinntao/Real-ESRGAN/blob/master/docs/Training.md
